### ISSUE TITLE
Can't pass tests on systems with locales having comma (",") as decimal point separator

### ISSUE THREAD

**Issue by tttomat19** [OpenHFT/Chronicle-Bytes#38]
Tests in error:
net.openhft.chronicle.bytes.BytesTest.testWriter[0](net.openhft.chronicle.bytes.BytesTest)
  Run 1: BytesTest.testWriter:605 ? InputMismatch

```
        @NotNull Scanner scan = new Scanner(bytes.reader());
        scan.useLocale(Locale.ENGLISH);
```

adding `scan.useLocale(Locale.ENGLISH);`
makes the test pass ok

https://upload.wikimedia.org/wikipedia/commons/thumb/a/a8/DecimalSeparator.svg/800px-DecimalSeparator.svg.png
green countries on the map actually use this comma separator by default.

**Comment by RobAustin** [comment]
thanks we added "scan.useLocale(Locale.ENGLISH);" to the test

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.101](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
